python/ml4ir/config/parse_args.py

```diff
@@ -214,6 +214,22 @@ def define_args() -> ArgumentParser:
         help="Gradient clipping value/threshold for the optimizer.",
     )
 
+    parser.add_argument(
+        "--compile_keras_model",
+        type=bool,
+        default=False,
+        help="Whether to compile a loaded SavedModel into a Keras model. "
```

Comment:
What does this mean, exactly?

Reply:
and what is the purpose?

Reply:
This is the key really. "Compile" means to be able to load a SavedModel file and then assign those weights to the RankingModel.model, which is a keras model. If you only want to run inference, you do not need to compile it. This was the "trick" to resave models. As a by-product it also allows us to retrain models.